esx_service/cli/vmdkops_admin.py

```diff
@@ -280,6 +280,22 @@ def commands():
                             }
                         },
 
+                        'replace': {
```

Comment:
I agree. Next time we will try to make this kind of change in a separate PR. This time since this PR is close to merge, I will keep as it is.